Chore/state of startups

MildTomato opened the pull request:
- updated header

MildTomato committed:
Add StateOfStartupsHeader component and update page

Introduces the StateOfStartupsHeader component for improved header styling on the state-of-startups page. Updates the page to use the new header and refactors the intro section layout. Adds a feature flag override for 'stateOfStartups' in useFlag for debugging purposes.

MildTomato committed:
Update StateOfStartupsHeader.tsx

MildTomato committed:
Update StateOfStartupsHeader.tsx

MildTomato committed:
Refactor progress bar components for reusability

Extracted a reusable ProgressBar component from DecorativeProgressBar and integrated it into StateOfStartupsHeader. Added support for configurable progress bars in header text blocks, improving code modularity and visual customization.

dnywh commented on apps/www/pages/state-of-startups.tsx:
```diff
@@ -206,10 +207,51 @@ function StateOfStartupsPage() {
+          <SurveySectionBreak />
+        </section>
+
+        {/* Original Header - Commented Out 
```
Happy for me to remove this in a follow-up PR?

dnywh reviewed (approved):
Just some minor suggestions. Looks good! Thanks for jumping on.

MildTomato committed:
Refactor StateOfStartupsHeader and update usage

Simplifies StateOfStartupsHeader by removing unused props and internalizing display logic. Progress bar delays are now randomized for a more dynamic effect. Updates state-of-startups.tsx to use the new header signature and removes legacy/commented code.

MildTomato committed:
Update StateOfStartupsHeader.tsx

MildTomato committed:
Update state-of-startups-og.png

MildTomato committed:
Update StateOfStartupsHeader.tsx

MildTomato committed:
Update header text color and OG image for State of Startups

Changed the 2025 TextBlock text color to 'text-brand-300' for improved visual consistency. Updated the Open Graph image for State of Startups.
